### ISSUE TITLE
Option to specify default Space

### ISSUE THREAD

**Issue by besendorf** [TandoorRecipes/recipes#2262]
### Is your feature request related to a problem? Please describe.

On my instance after creating the first space I noticed that every new user that registers for tandoor via LDAP automatically joins that space. Even after creating other spaces, new users still join the one space I created first. This is helpful for more use case as I want all users to join one space, however that would not work for every use case.
Im not sure how tandoor tandoor chooses the default space but I think that it is the one with the lowest id in the database.

### Describe the solution you'd like

A better solution would be to have an option to configure the default space.

**Comment by smilerz** [comment]
I'm actually not sure using LDAP in multispace setups is a good idea.

It's clear that you can already use that authentication method to break into the first space.  LDAP authentication isn't aware of Spaces at all and you'd need to overload default functions in that piece of functionality to make it space aware.

Testing to ensure that there was no leakage would also need to be fairly extensive.

**Comment by besendorf** [comment]
Is there an option for a single space setup? Or do you mean one should only use one space? This would not work as any user can create their own space

**Comment by smilerz** [comment]
> Is there an option for a single space setup? Or do you mean one should only use one space? This would not work as any user can create their own space

Hmm, I wasn't aware of that functionality. 

@vabene1111 LDAP and the current implementation of spaces aren't compatible.

**Comment by vabene1111** [comment]
you are correct in your observation, the default access joins you just into the first space

```python
user_space = UserSpace.objects.create(space=Space.objects.first(), user=request.user, active=False)
```

i kinda remembered that I added a setting to change the space a while ago but cant find it so I probably just wanted to do this. I think this option could be added, I would like to finish the discussion in #2261 to see what we come up there and maybe implement both in one go (and also documenting this along with every other auth flow)
